### PR TITLE
Add completions for builtin commands

### PR DESCRIPTION
**Pull request by deathaxe**
This commit adds completions for builtin project and workspace related commands.

### CODE REVIEW

File: plugins_/command_completions/builtin_commands_meta_data.yaml [hunk 1]
```diff
@@ -253,6 +280,12 @@ save:
 save_all:
   command_type: window
   doc_string: Save all the open documents in the active window.
+save_project_and_workspace_as:
+  command_type: window
+  doc_string: Save active project and worspace as ....
```
Comment (r-stein) [inline plugins_/command_completions/builtin_commands_meta_data.yaml]: Typo "worspace"

File: plugins_/command_completions/builtin_commands_meta_data.yaml [hunk 2]
```diff
@@ -253,6 +280,12 @@ save:
...
+save_workspace_as:
+  command_type: window
+  doc_string: Save active worspace as ....
```
Comment (r-stein) [inline plugins_/command_completions/builtin_commands_meta_data.yaml]: Same typo

File: plugins_/command_completions/builtin_commands_meta_data.yaml [hunk 3]
```diff
@@ -210,10 +223,21 @@ prompt_open_folder:
...
 prompt_save_as:
   command_type: window
   doc_string: Show the OS' native save as dialog, so the user can choose the path
     where they want the file to be saved.
+prompt_select_workspace:
+  command_type: window
+  doc_string: Quick Switch Project.
+prompt_switch_project_or_workspace:
+  command_type: window
+  doc_string: Show the OS' native save as dialog, so the user can choose what project they
```
Comment (keith-hall) [inline plugins_/command_completions/builtin_commands_meta_data.yaml]: shouldn't this be an "open" dialog?

Comment (deathaxe) [reply plugins_/command_completions/builtin_commands_meta_data.yaml]: Would make sense. ;-)

**Pull request merged**